tests/backup/backup_rancher_test.go

```diff
@@ -657,7 +670,7 @@ var _ = Describe("{MultipleProjectsAndNamespacesBackupAndRestore}", func() {
 			log.FailOnError(err, "Switching context to destination cluster failed")
 			for i := 0; i < 2; i++ {
 				destProject := fmt.Sprintf("dest-rke-project-%v-%v", RandomString(5), i)
-				_, err = Inst().S.(*rke.Rancher).CreateRancherProject(destProject, "new project", rancherActiveCluster)
+				_, err = Inst().S.(*rke.Rancher).CreateRancherProject(destProject, "new project", rancherActiveCluster, projectLabel, projectAnnotation)
```

Comment:
How about rancherProjectDescription instead of "new project"?

Reply:
done